### ISSUE TITLE
Multiple sequential newlines in JSDoc comments are not parsed correctly

### ISSUE THREAD

**Issue by Knagis** [Microsoft/TypeScript#8215]
**TypeScript Version:** 1.8.7

**Code**

``` ts
/** first line of the comment

third line */
function foo() {}
```

Note that there are no spaces on the empty line. If there were at least a single space, the parser would work as expected.

**Expected behavior:**

calling `Symbol.getDocumentationComment()` would return:

```
[
{ kind: "text", text: "first line of the comment" },
{ kind: "lineBreak" },
{ kind: "text", text: "" }, // this is optional
{ kind: "lineBreak" },
{ kine: "text", text: "third line" }
]
```

**Actual behavior:** 

```
[
{ kind: "text", text: "first line of the comment" },
{ kind: "lineBreak" },
{ kine: "text", text: "third line" }
]
```

**Cause**

[Line 532 of services/services.ts](https://github.com/Microsoft/TypeScript/blob/master/src/services/services.ts#L532) that consume all sequential newline characters.

**Fix**

I replaced locally this call to `consumeLineBreaks()` with this method:

```
function consumeOneLineBreak(pos, end, sourceFile) {
    if (pos < end) {
        var c = sourceFile.text.charCodeAt(pos);
        if (ts.isLineBreak(c)) {
            pos++;

        // consume \r\n as a single newline
            if (c === 13 && pos < end && sourceFile.text.charCodeAt(pos) === 10) {
                pos++;
            }
        }
    }
    return pos;
}
```

**Comment by billti** [comment]
@RyanCavanaugh to comment on the design/impl here. I believe the intent is to behave similarly to markdown (i.e. line breaks in the comment text purely for wrapping purposes are removed, but an empty line should be preserved).

**Comment by Knagis** [comment]
I am actually using the the API output to parse the comment text as markdown and this issue is preventing me from creating paragraphs.

**Comment by Knagis** [comment]
This seems to be fixed now in TypeScript 2.1 with the new jsdoc parser.

**Comment by sandersn** [comment]
Thanks for confirming @Knagis. I didn't know enough about the API to confirm that it was fixed.
